test/create.test.js

```diff
@@ -105,6 +106,23 @@ test('create more', (t) => {
   })
 })
 
+test('root message is encrypted', (t) => {
+  const alice = Testbot()
+
+  alice.tribes2.start()
+  alice.tribes2.create({}, async (err, group) => {
+    if (err) t.error(err, 'no error')
+
+    alice.db.getMsg(group.root, (err, kvt) => {
```

Comment:
@Powersource I made a commit which makes this simpler.

Yes I see this test passing in isolation too. I see the `master` branch is failing. I'd say this is safe to merge